Python/math/nth_term_of_GP.py

```diff
@@ -0,0 +1,39 @@
+import math
+'''
+This is a simple code in python to find the Nth term of a GP , let say 1 3 9 27 is GP its first term (a) is 1 and the Common Ratio (r) is 3 and if n=3 it means 3rd term which is 9.
```

Comment:
Justify align this and move it to the top.

Reply:
i have moved the statement to the top but  i don't understand justify align